pages/index.js

```diff
@@ -1,8 +1,24 @@
-import { gql, staticRequest } from "tinacms";
+import { staticRequest } from "tinacms";
 import { TinaMarkdown } from "tinacms/dist/rich-text";
 import { Layout } from "../components/Layout";
+import { useTina } from "tinacms/dist/edit-state";
+
+const query = `{
+  getPageDocument(relativePath: "home.mdx"){
+    data{
+      body
+    }
+  }
+}`;
+
 export default function Home(props) {
-  const content = props.data.getPageDocument.data.body;
+  const { data } = useTina({
```

Comment:
Maybe we could add a comment saying something along the lines of
`// data passes though in production mode and data is updated to be live data in edit mode`